pom.xml

```diff
@@ -1,42 +1,23 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
-
-    <groupId>com.aperto.magkit</groupId>
+    <parent>
+        <groupId>com.ibmix.magkit</groupId>
+        <artifactId>magkit-pom</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
     <artifactId>magnolia-vanity-url</artifactId>
     <version>1.6.1-SNAPSHOT</version>
     <packaging>jar</packaging>
     <name>magnolia-vanity-url Magnolia Module</name>
-
-    <inceptionYear>2013</inceptionYear>
```

Comment:
I think we should keep `inceptionYear` separate in each Magkit module, so let's not delete this here

Reply:
added again